docs/source/topics/packaging.rst

```diff
@@ -94,54 +95,82 @@ This directory structure is then zipped up and sent to AWS Lambda during the
 deployment process.
 
 
-Psycopg2 Example
-----------------
+Cryptography Example
+--------------------
+
+Below shows an example of how to use the
+`cryptography <https://pypi.python.org/pypi/cryptography>`__ package in a
+Chalice app for the ``python3.6`` lambda environment.
+
+We're going to leverage the ``vendor/`` directory in order to use this package
```

Comment:
What about just saying something like "Suppose we're on a Mac and we want to deploy an app that uses `cryptography`".  That keeps it more user-centric and less about implementation details of chalice building wheels.  You could also say that this wouldn't be an issue if we were on the same platform that Lambda uses.